packages/react/src/themes/teams/components/Embed/embedStyles.ts

```diff
@@ -0,0 +1,48 @@
+import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
+import { pxToRem } from '../../../../lib'
+import Embed, { EmbedProps, EmbedState } from '../../../../components/Embed/Embed'
+import { EmbedVariables } from './embedVariables'
+
+export default {
+  root: ({ props: p, variables: v }): ICSSInJSStyle => {
+    return {
+      boxSizing: 'border-box',
+      display: 'inline-block',
+      verticalAlign: 'middle',
+      position: 'relative',
+      cursor: 'pointer',
+      width: v.width,
+      height: v.height || 'auto',
+      outline: 0,
+
+      ...(p.isFromKeyboard && {
+        ':focus': {
+          [`& .${Embed.slotClassNames.control}`]: {
+            borderColor: v.focusBorderColor,
+            opacity: 1,
+          },
+        },
+      }),
+      ':hover': {
+        [`& .${Embed.slotClassNames.control}`]: {
+          opacity: 1,
+          zIndex: 1,
+        },
+      },
+    }
+  },
+  control: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    background: `0 no-repeat rgba(0,0,0,.25)`,
```

Comment:
Please let's use colors values in the variables, so that we may override those in different themes.